src/backup-all-indexes.sh

```diff
@@ -100,6 +109,9 @@ for index_name in $(curl -sS ${DATABASE_URL}/_cat/indices | grep logstash- | sed
       else
         echo "$(now): ${index_name} archival failed."
       fi
+      if [[ "${index_name:9}" > "${DELETE_CUTOFF_DATA}" ]]; then
```

Comment:
I'm fairly certain this should be `DELETE_CUTOFF_DATE`, not `DELETE_CUTOFF_DATA` (or this will delete all indices that are archive-able).

TBH, it's a real problem that this script does not run with `set -o nounset`, but for now, that's what it is :/